Don't use getopt for the command-line arguments.
It doesn't handle spaces within arguments correctly.

diff -Naurb xerces-c-src_2_8_0.orig/src/xercesc/runConfigure xerces-c-src_2_8_0/src/xercesc/runConfigure
--- xerces-c-src_2_8_0.orig/src/xercesc/runConfigure	2007-08-28 13:44:56.000000000 -0500
+++ xerces-c-src_2_8_0/src/xercesc/runConfigure	2010-03-16 15:23:10.000000000 -0500
@@ -114,17 +114,7 @@
 libtype=shared           # by default build shared libraries
 
 # Check the command line parameters
-if test -x /usr/bin/getopt -o -x bin/getopt; then
-	#
-	# os400 Users will need to comment out the next line.
-	getoptErr=`getopt p:c:x:dm:n:t:r:sb:l:z:P:C:h $*`
-	if [ $? != 0 ]
-	   then
-	   usage
-	   exit ${ERROR_EXIT_CODE}
-	fi
-	# Now get the command line parameters
-	set -- `getopt p:c:x:dm:n:t:r:sb:l:z:P:C:h $*`
+if true ; then
 	while [ $# -gt 0 ]
 	   do
 	   case $1 in
